webview/src/setup/components/studio/Studio.tsx

```diff
@@ -22,9 +27,28 @@ export const Studio: React.FC<{
     )
   }
 
+  const children = selfHostedStudioUrl && (
```

Comment:
[I] We should always show this section of the table so that users know the option is available.

When there is no URL set we can present "None" (or match what we show elsewhere on the setup page) in place of the URL.

Reply:
Good point! I'll update the section.